content/_getting-started/contracts-and-workspaces.md

```diff
@@ -110,19 +117,19 @@ client’s engineers, and production environment is customer-facing only.
 
 ## Limited Workspaces
 
-Workspaces can have the `limited` or `full` type. `Limited` is intended for platform trial period. Workspaces with `limited` type have certain restrictions for integration flows:
+The limited workspaces have certain restrictions for integration flows:
 
-- Integration flows in `limited` workspaces are restricted by work time. This means that they are automatically stopped after a certain time. This time is defined for each installation with an environment variable.
+*   Integration flows in the limited workspaces are restricted by work time. Meaning, the **platform automatically stops all active flows after 8 hours** in limited workspaces. You can restart the flows afterwards to continue your testing.
+*   If your integration flow contains messages in the processing queues, these messages will be removed when the time is up to stop the flow.
 
-- In `limited` workspaces, integration flows that contain errors will not be suspended. Instead, they will be stopped without saving any data.
+> We recommend not to use **limited workspaces** for the production purposes. Use **full workspaces** instead.
 
-Though not forbidden, **it is highly unrecommended to use `limited` workspaces for any production purposes**. `Limited` workspaces are indicated in the UI:
+The limited workspaces are clearly marked in the platform UI:
```

Comment:
* I would avoid the use of superfluous adjectives such as *clearly*, *obviously*, etc.

Reply:
ok